float64 battery_voltage
int8 rssi_dbm

